lib/board.rb

```diff
@@ -0,0 +1,45 @@
+class Board
+  def initialize
+    @board = [' ', ' ', ' ', ' ', ' ', ' ', ' ', ' ', ' ']
+    @selected_moves = []
+  end
+
+  def pick_a_square
+    loop do
+      square = gets.chomp.to_i - 1
```

Comment:
Please, `puts/gets` are not allowed in the `lib/` folder

Reply:
There is no puts or gets in the lib folder?